HW_23/generator.py

```diff
@@ -0,0 +1,23 @@
+import random
+import string
+
+def generator_of_unique_words(word_count: int):
+    if not (1 <= word_count <= 10_000):
+        raise ValueError("word_count must be between 1 and 10000")
+
+    unique_words = set()
+    word_lengths = range(3, 10)
+    words = string.ascii_lowercase
```

Comment:
should be normal words:
apple, dog, cat

You have combination of random symbols